packages/hash/integration/codegen.yml

```diff
@@ -1,5 +1,5 @@
 overwrite: true
-schema: ../api/src/graphql/typeDefs/*.ts
+schema: ../api/src/graphql/typeDefs/**/*.ts
```

Comment:
The integration tests generated GQL types were not picking up on all the GQL schemas, so have fixed that here.